src/app/order/hooks/usePostExistAddress.ts

```diff
@@ -0,0 +1,14 @@
+import { postExistAddress } from '@/apis/order';
+import { useMutation } from '@tanstack/react-query';
+
+const usePostExistAddress = () => {
+  const { mutateAsync, isLoading, error } = useMutation(postExistAddress);
```

Comment:
p2 : 위 코멘트에서 연결되는 부분인데, 새로운 배송지 등록 query가 Success되면 'existAddressList' 기존 배송지 조회 query Key를 재호출 하는 로직으로 변경하는 부분입니다.
```suggestion
 const queryClient = useQueryClient();
  const { mutateAsync, isLoading, error } = useMutation(postExistAddress, {
    onSuccess: () => {
      queryClient.invalidateQueries(['existAddressList']);
    },
  });
```